martinLi/assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js

```diff
@@ -22,31 +26,61 @@
 // Level Order Successor: 10
 
 class TreeNode {
-
   constructor(val) {
     this.val = val;
     this.left = null;
     this.right = null;
   }
-};
+}
 
-
-const find_successor = function(root, key) {
+const find_successor = function (root, key) {
   // TODO: Write your code here
-  return null;
-};
+  if (root == null) return null;
+
+  // If root equals to key
```

Comment:
Try to avoid inline comments as they clutter the code and make it more difficult to read.